todo.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+class Todo {
+  constructor() {
+    this.listTodo = [];
+    this.id = 0;
+    }
+
+  addItem(title, priority){
+    let item = {};
+    item.title = title;
+    item.priority = priority;    
+    item.id = this.id;
```

Comment:
instead of adding properties to an empty object, will be better to create an object with properties from start. For better readability, shorter code(less code -> fewer bugs), also it will be more performant
```javascript
const todo = {
  id: this.id,
  title,
  priority,
};
```